docs/cmdlets-reference/restore-cohesitymssqlobject.md

```diff
@@ -10,7 +10,7 @@ Restore-CohesityMSSQLObject -HostSourceId <long> -JobId <long> -SourceId <long>
  [-CaptureTailLogs] [-JobRunId <long>] [-KeepOffline] [-NewDatabaseName <string>] [-NewInstanceName <string>]
  [-RestoreTimeSecs <long>] [-StartTime <long>] [-TargetDataFilesDirectory <string>]
  [-TargetHostCredential <PSCredential>] [-TargetHostId <long>] [-TargetHostParentId <long>]
- [-TargetLogFilesDirectory <string>] [<CommonParameters>]
+ [-TargetLogFilesDirectory <string>] [-TargetSecondaryDataFilesDirectoryList <List<FilenamePatternToDirectory>>] [<CommonParameters>]
```

Comment:
What is the api param called ? IMHO this is a long variable name.

Reply:
The flag name is as is from the API. Its also self explanatory.